varnishgather

```diff
@@ -35,11 +35,15 @@ SECRET=""
 STATCMD=""
 ITEM=0
 TOPDIR=$(mktemp -d /tmp/varnishgather.XXXXXXXX)
-ID="$(hostname)-$(date +'%Y-%m-%d')"
+ID="$(cat /etc/hostname)-$(date +'%Y-%m-%d')"
 RELDIR="varnishgather-$ID"
 ORIGPWD=$PWD
-VERSION=1.75
+VERSION=1.76
 USERID=$(id -u)
+PID_ALL_VARNISHD=$(pidof varnishd 2> /dev/null)
+PID_ALL_VARNISHD_COMMA=$(pidof varnishd 2> /dev/null | sed "s/ /,/g")
+PID_VARNISHD=$(pgrep varnishd 2> /dev/null)
+PID_ALL_CACHEMAIN=$(pgrep cache-main 2> /dev/null)
```

Comment:
This should just be `PID_CACHEMAIN`. The `ALL` was supposed to mean parent and children. In this case, we have just `cache-main`.

Reply:
yup, makes sense. done!